sBTC SIP

```diff
@@ -0,0 +1,166 @@
+| sBTC Bootstrap Signer Set| The set of all sBTC signers. Each is registered with the .sbtc contract and the transfer. These entities as a group have full democratic access to the sBTC UTxO. |
+| sBTC Signer API          | API exposed by the sBTC Signer binary that handles basic low level commands. Remains private except to the Deposit API. |
+| Deposit API              | A third party API that communicates with the sBTC Bootstrap Signers via the sBTC Signer API. |
+
+## Problem Statement
+Bitcoin is limited in its programmability and scalability. While its security and censorship-resistance make it an ideal platform to build decentralized applications, it is not able to scale to meet the modern needs of users and developers. These challenges have been largely driven by the ethos of the Bitcoin ecosystem to prioritize simplicity and decentralization at the base layer.
+
+**Limitations with Bitcoin’s Scripting Language:** Bitcoin’s script has limited programmability, making it unsuitable for most decentralized applications. This leaves users with very few options to use financial applications, like lending and trading, without entrusting their Bitcoin to centralized entities. This also exposes users to counterparty risk, which has the potential to result in lost funds. This centralization contradicts the fundamental principles of Bitcoin and dampens the potential to unlock new functionality and scalability for users. These limitations in Bitcoin's script have made it challenging for developers to build applications directly on Bitcoin and led developers to seek alternative ecosystems that provide them with greater flexibility and tooling.
+
+**Limitations with Bitcoin’s Scalability:** Bitcoin is limited in its ability to process large amounts of data quickly and efficiently. Today, Bitcoin creates new blocks every 10 minutes on average, an interval which is longer than current state of the art blockchains like Ethereum (approximately 12 seconds) and Solana (about 400 milliseconds). This prohibits many types of applications that require faster confirmation times and creates challenges to provide good user experience. Additionally, during periods of high network congestion, this causes network transaction fees to increase, making it more expensive to transaction on the Bitcoin Layer 1 network.
+
+sBTC aims to solve these problems by enabling secure movement of BTC in and out of the Stacks network, providing fully decentralized and scalable Bitcoin applications. The goal is to implement a protocol that allows BTC holders to participate in smart contracts without relying on trusted third parties. By addressing these challenges, sBTC seeks to create new opportunities for Bitcoin, enhancing its utility and accessibility while preserving the core principles of decentralization and self-sovereignty.
```

Comment:
i'm not sure this should be part of the problem statement, rather it's indicative of a solution

Reply:
Yeah -- as a reader who's not already a participating developer, I'm not sure yet what the real problem is beyond "Bitcoin slow" (and we don't even solve that).

Reply:
If this text block is moved, I'd like to see an addition as to how this secure movment happens (ex: `secure movement of BTC in and out of the Stacks network via the sBTC smart contracts` or something similar)